src/components/DisplayEmbedTag/DisplayExternal.jsx

```diff
@@ -43,22 +45,42 @@ export class DisplayExternal extends React.Component {
         this.setState({ error: true });
       }
     } catch (e) {
+      handleError(e);
       this.setState({ error: true });
-      // throw new Error(e);
     }
   }
 
   render() {
-    const { onRemoveClick } = this.props;
+    const { onRemoveClick, url } = this.props;
     const { title, src, error, type, provider } = this.state;
 
-    if (!type && !provider) {
-      return null;
-    }
+    // TODO: When we need to support more, move this to helper function
+    const isH5p = url.indexOf('h5p') > -1;
+    const isYouTube = type === 'video' && provider === 'YouTube';
+
+    const externalIframe =
+      !isYouTube && !isH5p ? null : (
+        <iframe
+          style={
+            isYouTube
+              ? {
+                  minHeight: '436px',
```

Comment:
436?

Reply:
Optimal størrelse for youtube-videoer uten å få svarte kanter